### ISSUE TITLE
Create an icon

### ISSUE THREAD

**Issue by bojanz** [mglaman/conductor#4]
Do we use the Composer logo? Or ask someone to create us a unique one?

**Comment by mglaman** [comment]
I'd like a unique one. I'll whip up a draft.

**Comment by mglaman** [comment]
I'm thinking we can use https://thenounproject.com/search/?q=conductor&i=248063 in a circle/square. Just no idea on color choice.

**Comment by mglaman** [comment]
Linky link https://raw.githubusercontent.com/mglaman/conductor/master/build/icons/512x512.png

**Comment by joshmiller83** [comment]
I played around a bit this morning with the idea of an orchestra diagram. 

Inspired by this kind of image (lots of them on google images):

![image](https://cloud.githubusercontent.com/assets/1440242/19659976/0f66b9de-99fc-11e6-9279-acbaf887f609.png)

But didn't get too far. I'm thinking a half circle with a few interconnecting pieces and using shading and colors to provide rhythm. I'm happy to share the AI and PSD file of my attempt, but it is not even really ready for feedback, I just wanted to let you know which direction I took in case that might inform a future icon effort. Perhaps I'll pick up where I left off.

![image](https://cloud.githubusercontent.com/assets/1440242/19659958/0037a130-99fc-11e6-9bb8-f2703d8ff1c9.png)
